NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -14,31 +14,37 @@ public JsonResult OnGet()
         {
             List<Laureate> filteredLaureates = new List<Laureate>();
 
-            using (WebClient webClient = new WebClient())
-            {
-                string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
-                Prizes.Laureates Laureates = Prizes.Laureates.FromJson(PhysicsNinetyEight);
+            string physicsNinetyEightlaureates = getJsonFromUrl("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
+            Prizes.Laureates Laureates = Prizes.Laureates.FromJson(physicsNinetyEightlaureates);
```

Comment:
stay consistent here... if you're making one variable name lowerCamelCase, make them all lowerCamelCase.  Plus, since the class name is Laureates, the variable makes more sense as laureates than as Laureates... can be hard to distinguish when they are both capitalized the same way.